recipes-support/rauc-hawkbit/files/rauc-hawkbit.service

```
@@ -0,0 +1,9 @@
+[Unit]
+Description=rauc-hawkbit client service
+Requires=rauc.service
+
+[Service]
+ExecStart=/usr/bin/rauc-hawkbit-client -c /etc/rauc-hawkbit/config.cfg
```

To ensure flexibility, this should be

```suggestion
ExecStart=@BINDIR@/rauc-hawkbit-client -c @SYSCONFDIR@/rauc-hawkbit/config.cfg
```

with 

``` sh
sed -i -e 's!@BINDIR@!${bindir}!g' \
       -e 's!@SYSCONFDIR@!${sysconfdir}!g' ${D}${systemd_unitdir}/system/rauc-hawkbit.service
```

in do_install().

---

Done.